packages/data-mate/test/function-configs/function-configs-spec.ts

```diff
@@ -23,3 +25,38 @@ describe('function configs', () => {
         expect(names).toEqual(uniq(names));
     });
 });
+
+describe('function registries', () => {
+    const dirPath = path.join(__dirname, '..', '..', 'src', 'function-configs');
+
+    const configDirs = fs.readdirSync(dirPath);
+
+    for (const item of configDirs) {
+        // ignore non-directories
+        if (item.includes('.ts')) continue;
+
+        const functionPath = path.join(dirPath, item);
+
+        const imports = parseIndexFile(path.join(functionPath, 'index.ts'));
+
+        const configFiles = fs.readdirSync(functionPath).filter((i) => !(i.includes('utils') || i === 'index.ts'));
```

Comment:
We should use async functions here to not block the event loop

```suggestion
        const configFiles = fs.readdirSync(functionPath).filter((i) => !(i.endsWith('utils.ts') || i === 'index.ts'));
```